recipes/crcmod/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "crcmod" %}
+{% set version = "1.7" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "dc7051a0db5f2bd48665a990d3ec1cc305a466a77358ca4492826f41f283601e" %}
+
+package:
+    name: '{{ name|lower }}'
+    version: '{{ version }}'
+
+source:
+    fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+    '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+    number: 0
+    script: 'python setup.py install'
```

Comment:
Is there an issue with `python setup.py install --single-version-externally-managed --record record.txt`

Reply:
I thought this is just needed if setuptools is used.

Reply:
Ah that is most likely true, although it may also be required with noarch?